Makefile

```diff
@@ -54,7 +58,6 @@ minify: dist/rdflib.min.js
 writable:
 	@sed -i -re 's/git:\/\/github.com\//git@github.com:/' .git/config
 
-.PHONY: test
```

Comment:
I don't mind either way, but I thought the general Makefile convention was to place them right before their respective statements? (vs at the top)